genologics/entities.py

```diff
@@ -438,11 +439,43 @@ def create(cls, lims, container, position, **kwargs):
 
         Udfs can be sent in with the kwarg `udfs`. It should be a dictionary-like.
         """
+        instance = cls.create_in_memory_instance(lims, container, position, **kwargs)
+        data = lims.tostring(ElementTree.ElementTree(instance.root))
+        instance.root = lims.post(uri=lims.get_uri(cls._URI), data=data)
```

Comment:
I think here too that it would be more clear with in_mem_instance and created_instance.

Reply:
I'm not sure in this case, because it's first an in memory instance, but then we make it "more" than that by setting the root. Feels like I would have to rename more in that case.